packages/bot-web-ui/src/components/transaction.jsx

```diff
@@ -202,7 +207,7 @@ Transaction.propTypes = {
 };
 
 export default connect(({ transactions, run_panel }) => ({
-    contract_stage: run_panel.contract_stage,
     active_transaction_id: transactions.active_transaction_id,
+    contract_stage: run_panel.contract_stage,
```

Comment:
Missing prop type for `contract_stage`.

Reply:
Removed `contract_stage` since it's an unused prop.